### DISCUSSION TITLE
context.extensions.(...)user-agent

### DISCUSSION THREAD

**Discussion by sfraysse** [gaia-x-dases/xapi-virtual-classroom#6] [General]
It's very rare to find this kind of data into statements, which are supposed to give "instructional" information, more than "technical" information. This data is not included in the most common profiles such as video or CMI5.

So, I would let this kind of data out of the profile.

That does not mean that tools can't include this kind of data into their statements.
In fact, they can add extensions without breaking the conformance with the profile.

**Comment by sfraysse** [comment]
Sorry, in fact it is in the video profile, not in CMI5.
But I am still not convinced :)

**Comment by quitterie-lcs** [comment]
I agree and disagree with you but am still not be able to come to a conclusion too :sweat_smile: 

I would say it is as much useful that in the video profile for example for a dashboard about a given course and to have an overview about instructors activity. It would have the same utility to have an overview about the distribution of used browser to start classrooms. As for students activity with videos, which browser is more used to launch video ? 
But on the other side, an instructor dashboard is not at first sight in the scope of what we tend to define for this profile.

I agree with you, I would say, the less, the better but if the meantime if the information is not specified, people would not be aware to the possibility of tracking this information

**Comment by sfraysse** [comment]
One more thought on this topic...

When using tools such as MS Teams or Zoom, you may have a client installed on your laptop, so the virtual classroom in not launched in a browser. In this case, what should be the user agent? Is it allowed for installed clients to set their own value such as `Zoom/2.0` or something like that?

Anyway, we can let this point open for the moment and discuss it with a broader audience later...
